File: analyses/org.eclipse.tracecompass.incubator.scripting.ui.tests/scripts/perf/python/readTrace.py
loadModule("/TraceCompass/Trace")

trace = openMinimalTrace("Tracing", argv[0])

eventIterator = getEventIterator(trace)
schedSwitchCnt = 0;
while eventIterator.hasNext():
    event = eventIterator.next()
    if event.getName() == "sched_switch":
        schedSwitchCnt = schedSwitchCnt + 1

print(schedSwitchCnt)

File: analyses/org.eclipse.tracecompass.incubator.scripting.ui.tests/scripts/perf/javascript/readTrace.js
loadModule("/TraceCompass/Trace")

var trace = openMinimalTrace("Tracing", argv[0])

eventIterator = getEventIterator(trace);
schedSwitchCnt = 0;
while (eventIterator.hasNext()) {
    event = eventIterator.next();
    if (event.getName().equals("sched_switch")) {
        schedSwitchCnt = schedSwitchCnt + 1
    }
}
print("Sched switch " + schedSwitchCnt)